src/main/python/calibration/generate_rides_with_google_maps.py

```diff
@@ -0,0 +1,98 @@
+import pandas as pd
+import sys
+from pathlib import Path
+import glob
+
+
+def main(args):
+    if len(args) > 1:
+        input_file_path = Path(args[1])
+        if not Path.is_file(input_file_path):
+            print(f"The input file [{input_file_path}] does not exist")
```

Comment:
what about the inputs, e.g. sample size, distance, departure time range, etc.?

Reply:
The following parameters are supported now:
- sampleSize
- sampleSeed
- maxDistance
- departureTimeRange